superset-frontend/src/features/datasets/AddDataset/Footer/index.tsx

```diff
@@ -100,32 +107,57 @@ function Footer({
         if (typeof response === 'number') {
           logEvent(LOG_ACTIONS_DATASET_CREATION_SUCCESS, datasetObject);
           // When a dataset is created the response we get is its ID number
-          history.push(`/chart/add/?dataset=${datasetObject.table_name}`);
+          if (createChart) {
+            history.push(`/chart/add/?dataset=${datasetObject.table_name}`);
+          } else {
+            history.push('/tablemodelview/list/');
+          }
         }
       });
     }
   };
 
+  const onSaveOnly = () => {
+    onSave(false);
+  };
+
   const CREATE_DATASET_TEXT = t('Create dataset and create chart');
+  const CREATE_DATASET_ONLY_TEXT = t('Create dataset only');
```

Comment:
Not really the scope of this PR, but what do you think about?
**Create and explore dataset**
**Create dataset**
Wondering what are @yousoph and @kasiazjc's thoughts. Just to avoid "create" twice

Reply:
Yeah, I didn't want to change the original copy for now, but I'm happy to update this. :)

Reply:
That sounds good to me!